python_modules/dagster/dagster/daemon/sensor.py

```diff
@@ -565,7 +566,6 @@ def _get_or_create_sensor_run(
             # A run already exists and was launched for this time period,
             # but the daemon must have crashed before the tick could be put
             # into a SUCCESS state
```

Comment:
this comment seems very wrong too, remove? Maybe copied over from the scheduler where it is more correct. Unless i'm missing something this happens frequently even if nothing crashed